src/cluster/slot_migrate.cc

```diff
@@ -1130,3 +1148,164 @@ void SlotMigrator::resumeSyncCtx(const Status &migrate_result) {
     blocking_context_ = nullptr;
   }
 }
+
+Status SlotMigrator::sendMigrationBatch(BatchSender *batch) {
+  // user may dynamically change some configs, apply it when send data
+  batch->SetMaxBytes(migrate_batch_size_bytes_);
+  batch->SetBytesPerSecond(migrate_batch_bytes_per_sec_);
```

Comment:
```suggestion
  // Those configurations may be changed dynamically, so apply them here before sending.
  batch->SetMaxBytes(migrate_batch_size_bytes_);
  batch->SetBytesPerSecond(migrate_batch_bytes_per_sec_);
```

@caipengbo I cannot see those configurations will be changed except for constructing the migrator. Did I miss anything?

Reply:
`SetMigrateBatchSize` will be called in config set.

Reply:
oh, I missed this place. Thank you!